### DISCUSSION TITLE
Star sites

### DISCUSSION THREAD

**Discussion by kookma** [simonbaird/tiddlyhost#56] [Ideas]
It would be great if one can upvote /downvote (star and rank) a site on tiddlyhost.

**Comment by simonbaird** [comment]
I like the idea. It seems like it would add value to the hub if sites could be sorted by number of upvotes or likes.

**Comment by simonbaird** [comment]
Let's switch it to an issue.

**Reply by simonbaird** [reply]
https://github.com/simonbaird/tiddlyhost/issues/59
